scripts/fetch/fetch.gage_data.R

```diff
@@ -0,0 +1,15 @@
+fetch.gage_data <- function(viz = as.viz("gage-data")){
+  
+  depData <- readDepends(viz)
+  
+  sites <- depData[["nws-data"]][["sites"]]
+  config_data <- depData[["config"]]
+  dates <- depData[["dates"]]
+  
+  gage <- dataRetrieval::readNWISuv(sites$site_no, parameterCd = config_data$pCode,
+                            startDate = dates$startDate, endDate = dates$endDate)
+  
+  saveRDS(gage, file=viz[['location']])
+}
+
+fetchTimestamp.gage_data <- vizlab:::fetchTimestamp.file
```

Comment:
will this one need to get updated?

Reply:
and assuming yes, is it enough for us to only update when `depData[["dates"]]` changes?

Reply:
But the end of dates for that parameter block is often in the future (at least that is how we have been doing it), so I think this needs to fetch until that date is hit. And maybe continue to fetch, since some data may be altered after review.

Reply:
agreed. and agreed that a new issue to deal with this later (#22) is a good way to go.